### PR TITLE
Movie cards + Fix Footer Positioning and Update Trending Page Styling

### PR DESCRIPTION
**Pull request by Sisamkele022**
Description:
1. Footer Fix: Adjusted the footer position to ensure it stays at the bottom of the page when there is little content. The footer will now be pushed down, allowing the user to scroll when there is more content.
2. Trending Page Styling: Updated the styles for the trending page, including section titles, card layouts, and the glow effect for a more polished and consistent design. Added responsive design tweaks for better mobile compatibility.
3. Scrollbar Styling: Updated custom scrollbar for better user experience with more defined colors and hover effects.

### CODE REVIEW

File: src/utils/fetchMoviesByTitle.js [hunk 1]
```diff
@@ -0,0 +1,19 @@
+const API_KEY = "f4e3711c";
+const BASE_URL = "https://www.omdbapi.com/";
+
+export const fetchMoviesByTitle = async (title, type = "") => {
+  try {
+    const url = `${BASE_URL}?s=${title.replace(/ /g, "+")}&type=${type}&apikey=${API_KEY}`; // Use "s" for search
```
Comment (Lord-Nickelbottoms) [inline src/utils/fetchMoviesByTitle.js:6]: Remove unnecessary comment

File: src/utils/helpers.js
Comment (Lord-Nickelbottoms) [inline src/utils/helpers.js:1]: Should this file be empty?

**Review by LefaJele-Masemola — APPROVED** [review]
I see the changes of the footer but we still have to fix it!

**Review by SilindileShabangu — APPROVED** [review]
Great styling but please make space between the search bar and the heading

**Review by Ayanda-Radebe — APPROVED** [review]
The footers font should be changed for it to be more readable, also the header on home screen should be increased. The cards should have a hover colour

**Pull request merged**